lib/ansible/module_utils/vmware.py

```diff
@@ -66,6 +66,22 @@ def find_obj(content, vimtype, name, first=True):
     return [obj for obj in obj_list if obj.name == name]
 
 
+def find_objs(content, vimtypes, container=None, name=None):
```

Comment:
Could you please re-use `find_obj` method for this ?

Reply:
I can reuse the `find_obj` if you are ok with modifying it to accept a extra `container` (or `folder`) argument as in the `get_all_objs`.
Basically the `container or content.rootFolder` part is what I need to be able to limit the search (to a specific datacenter or cluster in the case of the resource_pool).

Reply:
There's also an inconsistency in the `find_obj` method, if `name=None` it ignores the `first` argument and always returns the first matching object or None.